### ISSUE TITLE
Avatars not stored in DB?

### ISSUE THREAD

**Issue by winterNebs** [Nandaka/PixivUtil2#748]
### Prerequisites

* [x] Did you read FAQ section?
* [zx] Did you test with the latest [releases](https://github.com/Nandaka/PixivUtil2/releases "latest releases") or [commit](https://github.com/Nandaka/PixivUtil2/commits/master "latest commit") ?

### Description

I use a image compression program to convert jpgs into .lep files, but every time i run the util it redownloads the avatars.

It would be nice to either have them in the db or have the option to ignore file extensions and only check names.

### Steps to Reproduce

1. Download avatars.
2. Remove/rename them
3. Try to download again

**Expected behavior:** Nothing happens

**Actual behavior:** It redownloads it 
### Versions
using latest 20200726

**Comment by Nandaka** [comment]
nope, as it can be updated anytime, also it has fixed filename not like image post.

**Comment by winterNebs** [comment]
Any suggestions for work arounds? Either making an option to ignore extensions or never update profile pictures after downloaded?

**Comment by Nandaka** [comment]
you can disable the avatar download in config.ini by setting `downloadavatar = False`
